### PR TITLE
Adds current limit methods

### PR DESCRIPTION
**Pull request by ManiacMoustache**
Tested, works with ctre/rev controllers. Adds dictionaries to doof config.

### CODE REVIEW

File: configs/lifter.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ lifter:
   motors:
     groups: "motors" 
     winchMotor:
-      channel: 40
+      channel: 41
```
Comment (ChrisHirsch) [inline configs/lifter.yml]: This is currently the wrong channel

File: utils/motorHelper.py [hunk 2]
```diff
@@ -42,35 +46,76 @@ def createMotor(motorDescp, motors = {}):
...
+        setREVCurrentLimits(motor, motorDescp)
 
     else:
         print("Unknown Motor")
 
     if 'inverted' in motorDescp:
         motor.setInverted(motorDescp['inverted'])
 
+
+    #if 'rampRate' in motorDescp:
```
Comment (ChrisHirsch) [inline utils/motorHelper.py]: If not needed this probably needs to be deleted

File: utils/motorHelper.py [hunk 3]
```diff
@@ -42,35 +46,76 @@ def createMotor(motorDescp, motors = {}):
...
+
+def setTalonFXCurrentLimits(motor, motorDescp):
+    """
+    Sets current limits based off of "currentLimits"
+    in your motor and config of choice. Must be a Talon FX motor controller
+    In currentLimits, you need currentLimit, triggerThresholdCurrent, and triggerThresholdTime.
+    """
+    if 'currentLimits' in motorDescp:
+        currentLimits = motorDescp['currentLimits']
+        currentLimit = currentLimits['currentLimit']
```
Comment (ChrisHirsch) [inline utils/motorHelper.py]: Isn't this circular?

Comment (ManiacMoustache) [reply utils/motorHelper.py]: No, inside currentLimits there is a key called currentLimit

**Comment by ChrisHirsch** [comment]
This PR currently crashes:
  File "Y:\Users\chirsch\src\raptacon\Robot-2020\utils\motorHelper.py", line 80, in setTalonFXCurrentLimits
    triggerThresholdCurrent = currentLimits['triggerThresholdCurrent']
KeyError: 'triggerThresholdCurrent'

**Pull request merged**